pkg/cmd/server/server_test.go

```diff
@@ -179,7 +179,8 @@ func TestParseCfg(t *testing.T) {
 			TableActor: &config.TableActorConfig{
 				EventBatchSize: 32,
 			},
-			EnableDBSorter: false,
+			EnableDBSorter:     false,
+			EnableNewScheduler: true,
```

Comment:
Could you remove `EnableNewScheduler` in debug.go ?

https://github.com/pingcap/tiflow/blob/92fc4c4593cafe3d81936de3f4eb5775683a8bae/pkg/config/debug.go#L35